jipdate.py

```diff
@@ -191,7 +196,7 @@ def get_jira_issues(jira, exclude_stories, epics_only, all_status, filename,
         f.write("# Header: %s\n" % issue.fields.summary)
         f.write("# Type: %s\n" % issue.fields.issuetype)
         f.write("# Status: %s\n" % issue.fields.status)
-        f.write("No updates since last week.\n\n")
+        f.write("# No updates since last week.\n\n")
```

Comment:
I think the idea is to strongly suggest to users to update ALL epics that are "In progress" and assigned to the user.  If there's nothing new, we should write that, because our members want an update on the epic every week, regardless :)

I think though, that the right way to tweak this beyond what it does today is to have config sections in a .config file somwhere, with the default text for a on update, the default intro etc.